contrib/tools/githubcopilot.mcp.yaml

```diff
@@ -0,0 +1,17 @@
+# Official MCP Server for GitHub Copilot
+# https://github.com/github/github-mcp-server
+apiVersion: kagent.dev/v1alpha1
+kind: ToolServer
+metadata:
+  name: api.githubcopilot.com
+  namespace: kagent
+spec:
+  config:
+    streamableHttp:
+      headers:
+        Authorization: "Bearer TOKEN"
```

Comment:
Rather than requiring the user to change this, can we instead use `fromSecret` added by @nimishamehta5.